index.js

```diff
@@ -337,7 +337,7 @@
                 /* Optimal order of selectors for checking. */
                 var selector_elements = ['year', 'holiday', 'month', 'monthday', 'week', 'weekday'];
                 for (var selector_ind in selector_elements) {
-                    if (selectors[selector_elements[selector_ind]].length > 0) {
+                    if (selector_elements.hasOwnProperty(selector_ind) && selectors[selector_elements[selector_ind]].length > 0) {
```

Comment:
Ok fine I'm pretty agree, I was just trying to comply with existing code formatting but yes it's not consistent everywhere.

So... in this case I suggest to make a PR first to put in place ESLint & fix existing code formatting, then rebase & push again on this PR, WDYT?